src/pages/Dashboard.tsx

```diff
@@ -4,27 +4,22 @@ import { addInsight, deleteInsight, getInsights } from '../actions/insightAction
 import { TEST_ORGANIZATION } from '@interfaces/organization';
 import { TEST_GROUP } from '@interfaces/group';
 import { Insight } from '@interfaces/insight';
-import { useEffect, useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
```

Comment:
I think I forgot to configure this in the linter but please remove unused variables